.gitignore

```diff
@@ -2,3 +2,4 @@ certs/
 savedfiles/
 settings.ini
 .env
+*.pyc
```

Comment:
Please explanation

Reply:
Added the '*.pyc' extension to gitignore.